features/sessions/src/test/java/com/android254/droidconKE2020/sessions/SessionsViewModelTest.kt

```diff
@@ -20,15 +21,20 @@ class SessionsViewModelTest : BaseViewModelTest() {
         sessionsViewModel = SessionsViewModel(sessionRepository)
     }
 
-    @Test
-    fun `test that sessions scheduled are fetched`() {
-        coEvery { sessionRepository.fetchSessionsSchedule("Day 1") } returns Data.Success(
-            testSessions
-        )
-        sessionsViewModel.fetchSessions("Day 1")
-        coVerify { sessionRepository.fetchSessionsSchedule("Day 1") }
-        sessionsViewModel.sessions.test().assertValue(testSessions)
-    }
+//    @Test
+// FIXME:
+// Here, I'd like to test that correct sessions were returned based on the `showBookmarked` boolean.
+// But I do not know how to do it. Someone please help me
+//    fun `test that sessions scheduled are fetched`() {
+//        coEvery { sessionRepository.fetchSessionsSchedule("Day 1") } returns Data.Success(
+//            testSessions
+//        )
+//        runBlocking {
+//            sessionsViewModel.fetchSessions(Pair(false, "Day 1"))
+//        }
+//        coVerify { sessionRepository.fetchSessionsSchedule("Day 1") }
+//        sessionsViewModel.filteredSessions.test().assertValue(testSessions)
+//    }
```

Comment:
@michaelbukachi @wangerekaharun please help me with this test.

Reply:
On it.

Reply:
@BKinya I've added the necessary test. I had to make some changes to your code.

Reply:
Thank you. I'll look at the changes.